superset-frontend/src/components/Card/index.tsx

```diff
@@ -28,11 +28,15 @@ const Card = ({ padded, ...props }: CardProps) => (
   <AntdCard
     {...props}
     css={(theme: SupersetTheme) => ({
+      // 'border-radius': `${theme.gridUnit}px`,
+      border: `1px solid ${theme.colors.grayscale.light2}`,
```

Comment:
```suggestion
```

Reply:
Oh, actually, there's probably a token we should be using for this!